packages/expansion-hub/src/open-rev-hub.ts

```diff
@@ -11,12 +11,52 @@ import {startKeepAlive} from "./start-keep-alive";
  */
 const openSerialMap = new Map<string, Serial>();
 
+/**
+ * Opens a parent Rev Hub. Does not discover or open any child hubs.
+ * If {@link moduleAddress} is not provided (or is undefined), it will take upwards of a second to
+ * learn the address of the parent hub.
+ *
+ * Call {@link ExpansionHub#addChildByAddress} to add known children. The serial number
+ * should start with 'DQ'.
+ *
+ * @param serialNumber
+ * @param moduleAddress
+ */
```

Comment:
```suggestion
/**
 * Opens a parent Expansion Hub. Does not open any child hubs.
 *
 * Call {@link ExpansionHub#addChildByAddress} to add known children.
 *
 * @param serialNumber The serial number of the Expansion Hub (should start with "DQ")
 * @param moduleAddress The module address of the parent (if this is not provided, it will take upwards of a second to
 * find the address of the parent hub).
 */
```

Reply:
This doc comment and the one for the next function also need to be brought into alignment, so that the only parts that are different are places where the functionality differs. Their similarities should be obvious, so that the user isn't second guessing if there are additional tradeoffs between them.

Reply:
Updated the docs